### PR TITLE
Unify settings and keybinds categories prefix

### PR DESCRIPTION
**Pull request by Dystopian**
**When merged this pull request will:**
- replace `ACE3 ` and `ACE -` prefixes and suffixes with `ACE ` prefix, as discussed in Slack.

### CODE REVIEW

**Comment by commy2** [comment]
I like this style very much.

File: addons/common/stringtable.xml [hunk 1]
```diff
@@ -999,94 +999,94 @@
...
         </Key>
         <Key ID="STR_ACE_Common_ACEKeybindCategoryVehicles">
-            <English>ACE3 Vehicles</English>
-            <German>ACE3 Fahrzeuge</German>
-            <Polish>ACE3 Pojazdy</Polish>
-            <Portuguese>Veículos ACE3</Portuguese>
-            <Russian>ACE3 Транспорт</Russian>
-            <Czech>ACE3 Vozidla</Czech>
-            <Spanish>ACE3 Vehículos</Spanish>
-            <Italian>Veicoli ACE3</Italian>
```
Comment (SilentSpike) [inline addons/common/stringtable.xml]: My only concern is whether this is no longer localised correctly?

Comment (jonpas) [reply addons/common/stringtable.xml]: Think of it as: `<mod> <category>`.

Comment (PabstMirror) [reply addons/common/stringtable.xml]: It's combining 2 nouns, I don't think the order could be that bad. 
("Equipment ACE3" sounds a little weird, but can be understood just fine)
Having the prefix first will be a big help for categories as they get sorted alphabetically and I think we want all of ace together.

**Pull request merged**